utils/readmp.f90

```diff
@@ -0,0 +1,46 @@
+!> @file
+!> @brief Read BUFR file containing embedded DX tables, and print
+!> each report one at a time
+
+program readmp
```

Comment:
Consider renaming to something that makes sense... read_**something**

Reply:
It should be `read_message`, ideally.

Reply:
how about bfdump? (witness ncdump)

Reply:
Also, as this is fortran code, it should contain doxygen documentation.

Reply:
done.